### PR TITLE
feat: add chakra ui

### PR DESCRIPTION
**Pull request by abrahamcalsin**
- [x] configure chakra ui
- [x] set dark theme
- [x] change some styles from tailwind to chakra ui
- [x] add styles for markdown (mdx)

### CODE REVIEW

File: pages/about-me/index.tsx [hunk 1]
```diff
@@ -11,32 +12,58 @@ const AboutMe = () => {
   return (
     <MainLayout>
       <Meta title="Sobre Mi" />
-      <div className="mt-0 sm:mt-11.2 sm:mb-3.7 text-primary-600 dark:text-inherit">
+      <Box mt={{ base: '0', sm: '90px' }} mb={{ base: '0', sn: '30px' }}>
```
Comment (yonycalsin) [inline pages/about-me/index.tsx:15]:
```suggestion
      <Box mt={{ base: '0', sm: '90px' }} mb={{ base: '0', sm: '30px' }}>
```

File: src/components/dark-theme-button/components/sun-fill-icon/sun-fill-icon.tsx [hunk 2]
```diff
@@ -23,8 +23,10 @@ export function SunFillIcon(props: SunFillIconProps) {
 
   return (
     <motion.svg
-      className={clsx('w-full dark:text-secondary', className)}
+      className={clsx('dark:text-secondary', className)}
```
Comment (yonycalsin) [inline src/components/dark-theme-button/components/sun-fill-icon/sun-fill-icon.tsx:26]: consider to remove the tailwind class

File: src/styles/globals.css [hunk 3]
```diff
@@ -5,19 +5,82 @@
 @tailwind components;
 @tailwind utilities;
 
+* {
+  /* outline: 1px solid rgba(200, 200, 0, 0.3) !important; */
+}
```
Comment (yonycalsin) [inline src/styles/globals.css:10]: remove this

**Review by yonycalsin — CHANGES_REQUESTED** [review]

File: pages/about-me/index.tsx (hunk 1 above)
Comment (abrahamcalsin) [reply pages/about-me/index.tsx:15]: ready, corrected, thanks 🙌

**Review by yonycalsin — APPROVED** [review]

**Pull request merged**